gremlin-go/driver/graphTraversalSource.go

```diff
@@ -163,3 +163,7 @@ func (gts *GraphTraversalSource) Io(args ...interface{}) *GraphTraversal {
 	traversal.bytecode.addStep("io", args...)
 	return traversal
 }
+
+func (gts *GraphTraversalSource) Tx() *transaction {
```

Comment:
This concerns me since a GTS is exported, and Tx is exported, but the return type is not.

Reply:
I think this needs to be exported

Reply:
done in https://github.com/lyndonbauto/tinkerpop/pull/68/commits/478457a4f6ce58124e732e2e2713a0cb32fca4de